### PR TITLE
Intel(R) oneAPI Collective Communications Library (oneCCL) 2021.14

### PR DESCRIPTION
**Pull request by nikitaxgusev**
This PR aligns the contents of the repository with the Intel(R) oneAPI Collective Communications Library 2021.14 release.

### CODE REVIEW

**Review by ranukund — APPROVED** [review]
Looks good! Thanks

File: CONTRIBUTING.md
Comment (Maria1Petrova) [inline CONTRIBUTING.md:1]: @maciekpac please check that contributing file is correct

**Review by i-kosarev — APPROVED** [review]
LGTM

File: CONTRIBUTING.md
Comment (maciekpac) [reply CONTRIBUTING.md:1]: it's not, changes are rolled back

File: MAINTAINERS.md
Comment (maciekpac) [inline MAINTAINERS.md:1]: changes were rolled back, they should not be - this file should remain unchanged

**Review by maciekpac — APPROVED** [review]
looks good now

**Review by Maria1Petrova — APPROVED** [review]
Looks good now

**Pull request merged**